feature/home/src/main/java/com/serranoie/app/feature/home/HomeScreen.kt

```
@@ -54,9 +51,107 @@ import androidx.navigation.compose.rememberNavController
+
+            else -> {
+                val daysRemaining = ChronoUnit.DAYS.between(today, end).toInt() + 1
+                TripDateInfo(
+                    status = TripStatus.ACTIVE,
+                    displayText = "$daysRemaining days left",
+                    subtitle = "$totalDays days total"
+                )
+            }
+        }
+
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)